R/tt_toString.R

```diff
@@ -296,23 +284,7 @@ setMethod(
       nrow = nrow(body),
       ncol = ncol(body)
     )
-    # Solve \n in titles
-    if (any(grepl("\n", all_titles(obj)))) {
-      if (any(grepl("\n", main_title(obj)))) {
-        tmp_title_vec <- .quick_handle_nl(main_title(obj))
-        main_title(obj) <- tmp_title_vec[1]
-        subtitles(obj) <- c(tmp_title_vec[-1], .quick_handle_nl(subtitles(obj)))
-      } else {
-        subtitles(obj) <- .quick_handle_nl(subtitles(obj))
-      }
-    }
-
-    # Solve \n in footers
-    main_footer(obj) <- .quick_handle_nl(main_footer(obj))
-    prov_footer(obj) <- .quick_handle_nl(prov_footer(obj))
```

Comment:
relevant modification (this was never meant to happen in matrix_form, but in the `MatrixPrintForm` constructor)